clang/docs/LanguageExtensions.rst

```diff
@@ -2761,6 +2761,47 @@ etc.).
 
 Query for this feature with ``__has_builtin(__builtin_assume_separate_storage)``.
 
+``__builtin_assume_dereferenceable``
+-------------------------------------
+
+``__builtin_assume_dereferenceable`` is used to provide the optimizer with the
+knowledge that the pointer argument P is dereferenceable up to the specified
```

Comment:
```suggestion
knowledge that the pointer argument P is dereferenceable at least up to the specified
```

Reply:
That has been adjusted a while ago, thanks!